db/db_service.go

```diff
@@ -94,9 +119,28 @@ func (svc *dbService) CreateApp(name string, pubkey string, maxAmountSat uint64,
 	svc.eventPublisher.Publish(&events.Event{
 		Event: "app_created",
 		Properties: map[string]interface{}{
-			"name": name,
+			"name":              name,
+			"id":                app.ID,
+			"walletChildPubkey": app.WalletChildPubkey,
 		},
 	})
 
 	return &app, pairingSecretKey, nil
 }
+
+func (svc *dbService) DeleteApp(app *App) error {
+
+	err := svc.db.Delete(app).Error
+	if err != nil {
+		return err
+	}
+	svc.eventPublisher.Publish(&events.Event{
+		Event: "app_deleted",
+		Properties: map[string]interface{}{
+			"name":              app.Name,
+			"id":                app.ID,
+			"walletChildPubkey": app.WalletChildPubkey,
```

Comment:
I'm not sure the wallet pubkey should be passed here. Instead, we can fetch it using the app ID if we need it (same with `app_created`)